WTWSMS/history/characters/WtWSMS_character_gift.txt

```diff
@@ -575,7 +575,7 @@
 	dynasty=1042303
 	religion="nicene"
 	culture="roman"
-	father=20787
+	father=20797
```

Comment:
I couldn't find info on this besides the incompatible death and birth dates, why should it be 20787?

Reply:
Because 20797 belongs to same dynasty, i think he was originally planned as a father for him and it was mistake in IDs.

Reply:
But 20797 is Paulinus, consul of 534?

Reply:
Yes

Reply:
He looks well as a father for Gaius Basilius.

Reply:
It is 20787 who should be his father.

Reply:
20787 (Caecina Decius Faustus Albinus) dies in 524, 31125 (Gaius Basilius) was born in 549, 20797 (Decius Paulinus) dies in 560, Albinus and Paulinus have very similar IDs, so i think this is ID mistake.

Reply:
Since 31125 is unused in title history we really can't know.

Reply:
I'm certain it isn't.

Reply:
But who is 31125? Why is 20787 meant to be his father? For me he looks like a fictional random no one whose job is only being someone's child.

Reply:
> But who is 31125? Why is 20787 meant to be his father? For me 31125 he looks like a fictional random no one whose job is only being someone's child.

He had an earlier job, I looked at an older version to find out this.

Reply:
I check in 0.9.3.1. First of all, the comment in titles he holds say "Basilius", not "Bailius". Small typo in character history makes him a lot harded to indentify 😠. I will fix it.

Reply:
Secondly, in 0.9.3.1 he holds d_senate in 550, but d_senate history in that version is... not very accurate... His consulship is fictional, correct me if I'm wrong.

Reply:
> Secondly, in 0.9.3.1 he holds d_senate in 550, but d_senate history in that version is... not very accurate... His consulship is fictional, correct me if I'm wrong.

Yes, that is why I said it had been fixed. I would make him be born and die  40 years earlier.

Reply:
> I would make him be born and die 40 years earlier.

This + mark him as fictional in character history, or we can just disable him to end his miserable life.

Reply:
But the most important thing is that 20787 probably wasn't put in his entry by accident, so the change proposed in the PR is not what we should do IMO.